tests/gtfs_reader.rs

```diff
@@ -18,6 +19,8 @@ fn load_minimal_agency() {
     let networks = gtfs::read_agency(tmp_dir.path());
     tmp_dir.close().expect("delete temp dir");
     assert_eq!(1, networks.len());
+    let agency = &networks.into_vec()[0];
```

Comment:
```rust
let agency = networks.iter().next().unwrap().1;
```
Minor for this case, but if you want to do something else with networks, `into_vec` will be problematic.